packages/builder/src/components/Header.tsx

```diff
@@ -49,6 +51,17 @@ export default function Header() {
               </Button>
             </Link>
             <ConnectButton />
+            <a
+              href="https://bridge.gitcoin.co"
+              target="_blank"
+              rel="noreferrer"
+              className="flex-shrink-0 flex items-center mx-3"
+            >
+              <span className="mr-2">
```

Comment:
Not an expert on tailwind but maybe you can use `gap-something` in `<a>` to separate icon and text instead of adding a span with margin.

Reply:
@bard I didn't find that but at least I removed the span wrapper and passed the margin directly to the svg element

Reply:
you can make the `<a>` a flex container, and then do `gap-x` where is a spacing value, but it feels a bit odd?